README.md

```diff
@@ -48,6 +48,7 @@ For more information, see [Why Score](https://score.dev/why-score).
   - team@score.dev
   - abuse@score.dev
 - See our [documentation](https://docs.score.dev).
+  - Build the [documentation site locally](content/en/README.md).
```

Comment:
I looked a few other docs repos and it seems to be quite common to display the content we show in `content/en/README.md` on the main readme as well. For example: https://github.com/krustlet/krustlet-docs, https://github.com/openservicemesh/osm-docs. I'm also generally not sure if the "What is Score", "Why Score" etc are relevant sections to be displayed on the docs repo. What do you think about simply linking to `content/en/README.md` from the main readme? E.g.:

Score Docs

Learn how to build the documentation site locally here.

Get involved
Contributing
Roadmap
License
Code of Conduct